### DISCUSSION TITLE
Need Mentorship

### DISCUSSION THREAD

**Discussion by RajputUsman** [coder/code-server#5373] [General] [answered]
Hi, Joe I am new to the opensource [@jsjoeio ](https://github.com/jsjoeio) contribution, I have done some minor pull requests. I saw you at **Open Source, TypeScript and Learning with Joe Previte** with James. I would love to learn and contribute. My exams still going through and I am eager to join the learning process will be free from 5 Aug.

**Comment by jsjoeio** [comment]
Hi @RajputUsman! Sorry I forgot to respond to this. Awesome :D

What is your experience with JS/TS and what types of contributions are you interested in? (docs, bugs, tests, etc)

**Comment by RajputUsman** [comment, marked as answer]
Thanks for the response, I have experience with JS and have built some react apps with it but have not worked with TS till now. I am more interested in bugs.

**Reply by jsjoeio** [reply]
Awesome! Well I suggest looking at the issues [here](https://github.com/coder/code-server/milestone/31) and commenting on one that you're interested in working on and we'll go from there. How does that sound?

**Reply by RajputUsman** [reply]
Awesome !!! 😎
